src/test/resources/test-rom/spec/programs/alias_spec.lua

```diff
@@ -0,0 +1,33 @@
+local capture = require "test_helpers".capture_program
+
+describe("The alias program", function()
+
+    it("displays its usage when given to many arguments", function()
+        
+        expect(capture(stub, "alias a b c"))
+            :matches { ok = true, output = "Usage: alias <alias> <program>\n", error = "" }
+    end)
+
+    it("list alias", function()
+        
+        expect(capture(stub, "alias"))
+            :matches { ok = true, output = "\n\n\n", error = "" }
+    end)
+
+    it("set alias", function()
+        shell.run("alias test Hello")
+
+        local tAlias = shell.aliases()
+
+        expect(tAlias.test):eq("Hello")
+    end)
+
+    it("clear alias", function()
+        shell.setAlias("test","hello")
```

Comment:
If we're doing this properly, you'd stub out bits of the shell API and verify that they were called with a specific method/arguments. This'll need some extensions to the test library though.